dags/blocksec_plugin/supertoken_downgrade_operator.py

```diff
@@ -1,11 +1,8 @@
-from airflow.models.baseoperator import BaseOperator
 from airflow.utils.decorators import apply_defaults
-from blocksec_plugin.web3_hook import Web3Hook
-from blocksec_plugin.ethereum_wallet_hook import EthereumWalletHook
+from blocksec_plugin.abis import DOWNGRADE_ABI
```

Comment:
Lets make a `SUPERTOKEN_ABI` using the ABI found here: 0xD6bd29da51D2875a6888C5d211b5Fb75bF28BA24 (USDCx supertoken contract address)